spec/indexers/section_indexer_spec.rb

```diff
@@ -0,0 +1,14 @@
+require 'rails_helper'
+
+describe SectionIndexer do
+  let(:indexer) { described_class.new(section) }
+  let(:monograph) { create(:monograph) }
```

Comment:
Instead of creating a monograph here, use a hard-coded ID:
`let(:monograph_id) { '123' }`